Adição de informações no Readme e ajustes do linter no Login.vue

ErisvaldoCorreia opened the pull request:
Tarefa Trello - FrontEnd (Adicionar Figma e Swagger no Readme)
Ajustes sugeridos pelo Linter na página Login.vue (nada na lógica foi alterado, apenas ajustes lints.

raffacabofrio commented on README.md:
```diff
@@ -1,4 +1,6 @@
-# NewSchool-frontend
+# NewSchool-frontend  
+![new school version](https://img.shields.io/badge/Version_of_Project-1.0.0-green)
+![nuxt version](https://img.shields.io/badge/Nuxt-%5E2.0.0-yellow)
```
Favor remover versão do Readme. Vamos versionar pelo releases do Github.

Mas seria legal ter um chagelog com link pro releases. 
https://github.com/NewSchoolBR/newschool-frontend/releases

ErisvaldoCorreia replied:
Maravilha vou ajustar isso então, sem problemas!

raffacabofrio commented on README.md:
```diff
@@ -71,4 +73,12 @@ Depois de concluir e testar, envie e aguarde o PULL REQUEST ser aprovado.
+
+http://newschoolbrapi-dev.herokuapp.com/swagger/#/default 
+
+CONHEÇA O PROJETO NO FIGMA: 
```
Eu trocaria por CONFIRA OS LAYOUTS NO FIGMA

ErisvaldoCorreia replied:
Certo... vou ajustar!

ErisvaldoCorreia committed:
Alterações solicitadas pelo Raffa

Remoção das Tags de versão, deixando um link para changelog dos releases.

Mudança no cabeçalho de chamada do Figma.

ErisvaldoCorreia commented:
Alterações solicitadas incluidas! :)

filipefelice reviewed (approved):
Boa Junior! Isso ae ;)
👏

ErisvaldoCorreia committed:
Adição de informações no Readme e ajustes do linter no Login.vue para aspas simples

wantero commented on pages/login.vue:
```diff
@@ -43,69 +41,69 @@
-      v => !!v || "Digite a senha",
-      v => (v && v.length >= 6) || "A senha deve ter no mínimo 6 caractéres"
+      v => !!v || 'Digite a senha',
+      v => (v && v.length >= 6) || 'A senha deve ter no mínimo 6 caractéres'
```
@ErisvaldoCorreia não é referente a sua PR, mas já poderia corrigir a palavra caracteres? A mesma não possui acento.

raffacabofrio replied:
Boa. Fazendo isso podemos aprovar e mergear.

andrews-felipe replied:
essa alteração foi inclusa na minha PR

wantero replied:
show! aprovando essa então.

raffacabofrio commented:
@ErisvaldoCorreia Está com conflito.

Eu sugiro vc voltar package-lock.json para ficar igual a develop. E atualizar sua PR.

ErisvaldoCorreia committed:
Sincronizando - e Reajustando o Readme

ErisvaldoCorreia committed:
Merge branch 'develop' of https://github.com/ErisvaldoCorreia/newschool-frontend into develop
